### ISSUE TITLE
Possible to replace editor with notepad++?

### ISSUE THREAD

**Issue by namoran** [sabrogden/Ditto#739]
I like the app, but when I edit a clip I would like to use notepad++ but I can't figure out how to do it. I thought maybe I could launch np++ with the chaiscript when text is copied but then how will I get the text back to replace the clip when finished editing? Not sure. Any ideas?

**Comment by sabrogden** [comment]
Currently no but the editor is outdated and doesn't work with screen scaling.  So maybe in the future.

**Comment by ALERTua** [comment]
yeah, that's cool to have, but I have no .rtf editors, and you took the only way to edit the clips from me :)

**Comment by Chealer** [comment]
@namoran: This is not possible except for versions 3.25.73.0 to 3.25.76.0 (the change was reverted in 3.25.113). Ticket #550 basically asks the same.
